cli/command/stack/deploy.go

```diff
@@ -520,6 +528,45 @@ func convertService(
 	return serviceSpec, nil
 }
 
+func convertHealthcheck(healthcheck *composetypes.HealthCheckConfig) (*container.HealthConfig, error) {
+	if healthcheck == nil {
+		return nil, nil
+	}
+	var (
+		err               error
+		test              []string
+		timeout, interval time.Duration
+		retries           int
+	)
+	// FIXME(vdemeester) decide what mean NONE for healthcheck
+	if len(healthcheck.Command) == 0 || healthcheck.Command[0] == "none" {
+		test = []string{"NONE"}
```

Comment:
I'd not use a magic value.
Something like `DisableHealthCheck: true` seems good to me.
Then someone can just add or remove this to the existing config to enable/disable.